v4/source/system.yaml

```diff
@@ -1488,3 +1488,54 @@
           $ref: "#/components/responses/Forbidden"
         "404":
           $ref: "#/components/responses/NotFound"
+  /system/first_admin_visit_marketplace:
+    get:
+      tags:
+        - system
+      summary: Retrieves the system-level status that specifies that at least an admin has visited the in-product Plugin Marketplace.
```

Comment:
```suggestion
      summary: Retrieves the system-level status that specifies that at least an Admin has visited the in-product Plugin Marketplace.
```

Reply:
One question: When we say "at least an Admin" do we mean "at the very least, an Admin visited the Marketplace" or "at least one Admin visited..." or something else? The sentence is a little unclear in that respect.

Reply:
@justinegeffen - "at least one Admin visited..." - or in other words, the first one to visit the in-product Plugin Marketplace turns the badge off.

Reply:
Perfect, thank you!